javascript/javascript1/week3/item_removal.js

```diff
@@ -0,0 +1,8 @@
+//removing an element of an array using splice method
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+const nameToRemove = 'Ahmad';
+ 
+names.splice(names.indexOf(nameToRemove), 1);
```

Comment:
almost right. It could be that the name is not found, and you need to handle that case.